### ISSUE TITLE
[Reports] Orders And Distributors report is showing Order ID instead of Order Number

### ISSUE THREAD

**Issue by drummer83** [openfoodfoundation/wishlist#481]
## Description

The Orders And Distributors report is showing the Order ID instead of Order Number. Other reports use the order number because the order ID is not very helpful for users.

## Expected Behavior

The report should display the order number instead of the order ID.

## Actual Behaviour

The report is displaying the order ID instead of the order number.

## Steps to Reproduce

1. Run the Orders And Distributors report.

## Animated Gif/Screenshot

Here is an example:
![image](https://github.com/openfoodfoundation/wishlist/assets/1790176/9f9e94ff-e399-482f-833c-8f271b65a841)

## Workaround

None.

## Severity
bug-s4

## Your Environment

* Version used: staged #11543 
* Browser name and version:
* Operating System and version (desktop or mobile):

**Comment by RachL** [comment]
@lin-d-hop it looks like this is something that was designed that way - do you remember why?

**Comment by lin-d-hop** [comment]
I dont think it was designed this way, but I think we didn't change it because order id meant that we could use it in integrations.

If we want full consistency on this we would need to add order number hidden, then swap so id is hidden and number is shown. It makes sense theoretically to eventually remove id from reports.... but I'm not sure in practice if that makes sense currently.
